Shared.Tests/UtilsTest.cs

```diff
@@ -80,5 +80,17 @@ public void TestEncodeUrlSafeBase64String()
 
             Assert.AreEqual("YWQ_Lix4MDl-IUAhYQ==", Utils.EncodeUrlSafe("ad?.,x09~!@!a"));
         }
+
+        [Test]
+        public void TestComputeHexHash()
```

Comment:
Please add reference input values and expected result. In case implementation is broken, these tests will still pass.

Reply:
fixed